docs/reference/commandline/events.md

```diff
@@ -165,40 +165,39 @@ The filtering flag (`-f` or `--filter`) format is of "key=value". If you would
 like to use multiple filters, pass multiple flags (e.g.,
 `--filter "foo=bar" --filter "bif=baz"`)
 
-Using the same filter multiple times will be handled as a *OR*; for example
-`--filter container=588a23dac085 --filter container=a8f7720b8c22` will display
-events for container 588a23dac085 *OR* container a8f7720b8c22
+Using the same filter multiple times is interpreted as a logical `OR`; for example,
+`--filter container=588a23dac085 --filter container=a8f7720b8c22` displays
+events for container `588a23dac085` or container `a8f7720b8c22`.
 
-Using multiple filters will be handled as a *AND*; for example
-`--filter container=588a23dac085 --filter event=start` will display events for
-container container 588a23dac085 *AND* the event type is *start*
+Using multiple filters is interpreted as a logical `AND`; for example,
+`--filter container=588a23dac085 --filter event=start` displays events for
+container `588a23dac085` and where the event type is `start`.
 
 The currently supported filters are:
 
-* config (`config=<name or id>`)
-* container (`container=<name or id>`)
-* daemon (`daemon=<name or id>`)
-* event (`event=<event action>`)
-* image (`image=<repository or tag>`)
-* label (`label=<key>` or `label=<key>=<value>`)
-* network (`network=<name or id>`)
-* node (`node=<id>`)
-* plugin (`plugin=<name or id>`)
-* scope (`scope=<local or swarm>`)
-* secret (`secret=<name or id>`)
-* service (`service=<name or id>`)
-* type (`type=<container or image or volume or network or daemon or plugin or service or node or secret or config>`)
-* volume (`volume=<name>`)
+- config (`config=<name or id>`)
+- container (`container=<name or id>`)
+- daemon (`daemon=<name or id>`)
+- event (`event=<event action>`)
+- image (`image=<repository or tag>`)
+- label (`label=<key>` or `label=<key>=<value>`)
```

Comment:
Reminds me I had a PR to rewrite this to use tables, but some of the combinations in my examples were wrong, as combining some did not produce the same as `--type` 🤔 (IIRC); perhaps you're interested in carrying that PR though;

- https://github.com/docker/cli/pull/3263

Reply:
Let me carry this one, but I'll do it in a follow-up as not to block this one since there's an open technical question